src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1870,250 +3129,379 @@
     short-summary: Update a peering.
-    long-summary: To learn more about updating virtual network peerings, visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering
     examples:
-        - name: Change forwarded traffic configuration of a virtual network peering
+        - name: Change forwarded traffic configuration of a virtual network peering.
           text: >
-            az network vnet peering update ---name MyVnet1toMyVnet2 --resource-group MyResourceGroup --vnet-name MyVnet1 --set allowForwardedTraffic==true
-        - name: Change virtual network access of a virtual network peering
+            az network vnet peering update -g MyResourceGroup -n MyVnet1ToMyVnet2 --vnet-name MyVnet1 --set allowForwardedTraffic=true
+        - name: Change virtual network access of a virtual network peering.
           text: >
-            az network vnet peering update ---name MyVnet1toMyVnet2 --resource-group MyResourceGroup --vnet-name MyVnet1 --set allowVirtualNetworkAccess==true
-        - name: Change gateway transit property configuration of a virtual network peering
+            az network vnet peering update -g MyResourceGroup -n MyVnet1ToMyVnet2 --vnet-name MyVnet1 --set allowVirtualNetworkAccess=true
+        - name: Change gateway transit property configuration of a virtual network peering.
           text: >
-            az network vnet peering update ---name MyVnet1toMyVnet2 --resource-group MyResourceGroup --vnet-name MyVnet1 --set allowGatewayTransit==true
-        - name: Use remote gateways in virtual network peering
+            az network vnet peering update -g MyResourceGroup -n MyVnet1ToMyVnet2 --vnet-name MyVnet1 --set allowGatewayTransit=true
+        - name: Use remote gateways in virtual network peering.
           text: >
-            az network vnet peering update ---name MyVnet1toMyVnet2 --resource-group MyResourceGroup --vnet-name MyVnet1 --set useRemoteGateways==true
+            az network vnet peering update -g MyResourceGroup -n MyVnet1ToMyVnet2 --vnet-name MyVnet1 --set useRemoteGateways=true
 """
-
 # endregion
 
 # region VPN Connection
 helps['network vpn-connection'] = """
     type: group
     short-summary: Manage VPN connections.
+    long-summary: >
+        For more information on site-to-site connections,
+        visit https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-site-to-site-resource-manager-cli.
+        For more information on Vnet-to-Vnet connections, visit https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-vnet-vnet-cli
 """
 
 helps['network vpn-connection create'] = """
     type: command
     short-summary: Create a VPN connection.
+    examples:
+        - name: >
+            Create a site-to-site connection between an Azure virtual network and an on-premises local network gateway.
+            NOTE: The VPN Gateway and Local Network Gateway must be provisioned before creating the connection between them.
```

Comment:
Remove `NOTE` and everything after it.

Reply:
done. Removed "note", moved remaining to long-summary.